README.md

```diff
@@ -101,7 +101,7 @@ Note that the package `clang` includes `libclang` as well as the C++ compiler.
 Once the dependencies are in place, you can build Zebra
 
 ```sh
-cargo install --locked --git https://github.com/ZcashFoundation/zebra --tag v1.0.0-rc.8 zebrad
+cargo install --locked --git https://github.com/ZcashFoundation/zebra --tag v1.0.0-rc.9 zebrad
 ```
 
 You can start Zebra by
```

Comment:
Note: remove the "Output of `help`, `--help`" known issue after PR #6801 merges.

Reply:
Done in e5c6bd38